### DISCUSSION TITLE
When I input a semicolon (:), the content before the semicolon disappears.

### DISCUSSION THREAD

**Discussion by NoelYoon96** [i18next/next-i18next#2298] [General]
For example, when I input '0:30(Chorus)', it gets rendered as '30(Chorus)'. Even if I put '0:30(Chorus)' as a key in the JSON file located in the locales directory, it still gets rendered as '30(Chorus)'. Could this be a potential issue?

**Comment by adrai** [comment]
provide a reproducible example repository showing your issue
